consensus/config/src/parameters.rs

```diff
@@ -131,12 +131,16 @@ impl Parameters {
         200
     }
 
-    pub(crate) fn default_sync_last_proposed_block_timeout() -> Duration {
-        Duration::ZERO
+    pub(crate) fn default_sync_last_known_own_block_timeout() -> Duration {
+        if cfg!(msim) {
+            Duration::from_millis(200)
+        } else {
+            Duration::from_secs(5)
+        }
     }
 
-    pub fn is_sync_last_proposed_block_enabled(&self) -> bool {
-        !self.sync_last_proposed_block_timeout.is_zero()
+    pub fn sync_last_known_own_block_enabled(&self) -> bool {
```

Comment:
nit: since this is an interpretation of the config, maybe this check should be done at the callsite.